Wangscape/AlphaCalculator.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <vector>
+#include <functional>
+#include <SFML/Config.hpp>
+class AlphaCalculator
+{
+public:
+    typedef std::vector<float> WeightVector;
+    typedef std::vector<sf::Uint8> AlphaVector;
```

Comment:
Yes, `using` statements for some of the most common names from `std` would be reasonable.